humilis_sam/outputs.yaml.j2

```diff
@@ -1,5 +1,19 @@
 ---
 outputs:
+    {% if swagger_template %}
+    RestApiId:
```

Comment:
The API key is not attached to the deployed API because that doesn't seem to work using CF. We will probably have to create a custom CF resource to take care of that at some point. For now the key will have to be attached manually to the API stage using an _after deployment_ script.